components/Home/Bonding/requests.jsx

```diff
@@ -248,35 +248,42 @@ const getProductDetailsFromIds = ({ productIdList }) => new Promise((resolve, re
 
   try {
     const allListPromise = [];
+    const allListCurrentPriceLpPromise = [];
 
     for (let i = 0; i < productIdList.length; i += 1) {
       const id = productIdList[i];
-      const result = contract.methods.mapBondProducts(id).call();
-      allListPromise.push(result);
+      const allListResult = contract.methods.mapBondProducts(id).call();
+      const allListCurrentPriceLpResult = contract.methods.getCurrentPriceLP('0x09D1d767eDF8Fa23A64C51fa559E0688E526812F').call();
```

Comment:
Todo – hard-coded this in a hurry – @mohandast52 please return to this when you can and update using dynamic token data.